autoPyTorch/pipeline/base_pipeline.py

```diff
@@ -408,7 +408,7 @@ def get_default_pipeline_options() -> Dict[str, Any]:
             'job_id': '1',
             'device': 'cpu',
             'budget_type': 'epochs',
-            'epochs': 20,
+            'epochs': 1,
```

Comment:
I assume this was for testing? We probably want 20 as default

Reply:
I think we probably want all of this in another file where to read the defaults in general

Reply:
Yeah, probably autoPyTorch/configs would be most suitable, we can also store search space presets there